docs/news/posts/2024-10-04-data-access.md

```diff
@@ -0,0 +1,226 @@
+    project=[orm.Group.fields.label],
+    tag="group",
+)
+qb.append(
+    orm.Int,
+    with_incoming="group",
+    filters=(orm.Int.fields.pk > 1) & (orm.Int.fields.pk < 10),
+    project=[orm.Int.fields.pk, orm.Int.fields.value],
+)
+```
+
+Through this, for example the filter on the values of the `pk`s reduces from:
+
+```python
+filters={
+    "and": [
+        {"pk": {">": 1}},
+        {"pk": {"<": 10}},
+    ]
+}
+```
+
+to the more concise:
+
+```python
+filters=(orm.Int.fields.pk > 1) & (orm.Int.fields.pk < 10),
+```
+
+in which the `"and"` condition can be expressed via `&` and applied on the relevant entities. Furthermore,
+accessing through the `.fields` attribute, e.g. in the updated `project`ion:
+
+```python
+project=[orm.Int.fields.pk, orm.Int.fields.value]
+```
+
+albeit being more verbose, is less error-prone than the access via string identifiers in the previous version:
+
+```python
+project=["pk", "attributes.value"]
+```
+
+as it allows for autocompletion.
+
+Any feedback on the new QB syntax welcome!
+
+***
+
+## Relevant PRs
+
+For the more tech-savvy among us, here are the relevant PRs of the changes outlined in this blog post:
+
+- [[#6276]](https://github.com/aiidateam/aiida-core/pull/6276) Add CLI command to dump inputs/outputs of `CalcJob`/`WorkChain`
+- [[#6245]](https://github.com/aiidateam/aiida-core/pull/6245) ✨ NEW: Add `orm.Entity.fields` interface for `QueryBuilder`
+  (cont.) [and linked PRs]
+
+## Footnotes
+
+[^1]:
+    The file repository is based on the [`disk-objectstore`](https://github.com/aiidateam/disk-objectstore)
+    implementation. If you ever wondered what the `_dos` appendix of the `core.psql_dos` and `core.sqlite_dos` storage
+    backends means, now you know! 😉
+
+[^2]:
+    While your workflows are being executed on the remote computer, files are located in a three level hierarchical
+    directory structure under the `work_directory` of the given `Computer` (typically in the `scratch`), with folder names
+    based on the universally unique identifiers (UUIDs) of the corresponding AiiDA `CalculationNode`s. For instance, if
+    the UUID of your `CalculationNode` is `6861d8fb-4694-46be-b0e6-7282989f069d`, your files will be in
+    `68/61/d8fb-4694-46be-b0e6-7282989f069d`. Tip: The `verdi calcjob gotocomputer <pk>` command provides a simple way to enter a
+    shell session in the remote working directory of your calculation.
+
+[^3]:
+    Which actually contains the previous `PwCalculation` as one of its steps.
+
+[^4]:
+    The workflow is recursively traversed, and files are written to disk for each calculation. The subdirectory naming is
+    automatically determined and based on the iteration counter, the link label, and the class name.
+
+[^5]:
+    The `dump`ing feature is currently still actively developed to enable obtaining remote and stashed data entities (e.g., intermediate files of the workflow that weren't originally retrieved from the HPC), as well as allowing for
```

Comment:
As a general comment, I don't know if using backticks as part of a longer work makes things readable. I would just write

The dumping feature

or

The `verdi process dump` feature

or e.g.

The dumping feature (via `verdi process dump`)

but avoid e.g. `dump`ing or similar - if you agree, I would change also the rest